packages/infra/src/constructs/lambdaDeployment.ts

```diff
@@ -61,5 +72,19 @@ export class LambdaDeployment extends Construct {
         'aws-codedeploy-hook-AfterAllowTraffic',
       ),
     );
+
+    if (!containsSkipDirective(buildMessage, 'alarm')) {
+      deploymentGroup.addAlarm(
+        new aws_cloudwatch.Alarm(this, 'codedeploy-alarm', {
```

Comment:
Is consistency of any value here:

```suggestion
        new aws_cloudwatch.Alarm(this, 'CodeDeployAlarm', {
```